pages/GetAccountLists.graphql

```diff
@@ -1,4 +1,8 @@
 query GetAccountLists {
+  user {
+    id
+    setup
```

Comment:
Did you intentionally use `setup` from `user` rather than getting the `userOptions`? I guess all we need now is a boolean.

Reply:
So `user.setup` is generated by the server and tells us whether they have no account lists, no organization accounts, or no default account chosen. If it is not null, they need to go through the setup screens I made. After that phase, we set `setup_position` and take the user through the tour screens you made. Does that make sense?

Reply:
Ah ok, that makes sense. I didn't realize what `user.setup` was.

Reply:
It's definitely confusing